test/test_command_line_interface.py

```diff
@@ -210,6 +210,27 @@ def test_postaddr_lists_only_contacts_with_post_addresses(self):
                   '                             SomeState, HomeCountry']
         self.assertListEqual(expect, text)
 
+    def test_mixed_kinds(self):
+        with TmpConfig(["org.vcf", "individual.vcf"]):
```

Comment:
Did you commit these files? I don't see them.

Reply:
I forgot to add them… 🤦‍♂️